ibexa/commerce/5.0/config/packages/security.yaml

```diff
@@ -83,8 +83,8 @@ security:
         # Uncomment `ibexa_cdp` rule if you are using Ibexa CDP connector
         #ibexa_cdp:
         #    pattern: /cdp/webhook
-        #    guard:
-        #        authenticator: 'Ibexa\Cdp\Security\CdpRequestAuthenticator'
+        #    custom_authenticators:
+        #        - 'Ibexa\Cdp\Security\CdpRequestAuthenticator'
```

Comment:
Does this endpoint work in context of other firewalls? Can this be accessed with a different authentication that `CdpRequestAuthenticator` (like currently logged in user)?

If no, then this should probably have (and be) it's own [entrypoint](https://symfony.com/doc/current/security/entry_point.html)

Reply:
According to Symfony doc:
> [!NOTE]
> Starting with Symfony 5.2, the custom authenticator is automatically registered as an entry point if it implements AuthenticationEntryPointInterface.
> Prior to 5.2, you had to configure the entry point separately using the entry_point option. Read [The Entry Point: Helping Users Start Authentication](https://symfony.com/doc/5.x/security/entry_point.html) for more information.